frontend/src/components/WofInfoBox.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <wof-popper>
+    <table :class="`WofBox WofBox--${this.type}`">
+        <tr :id="`first-row`">
+            <th :id="`title`">{{ title }}</th>
+            <th :id="`close-window`">
+                <wof-button :type="this.type" :id="`close-window-button`" @click="$emit('close')">
+                <wof-icon icon="wof-cancel" size=2  />
+                </wof-button>
+            </th>
+        </tr>
+        <tr>
+        <th colspan="2" id="content"><slot></slot></th>
+        </tr>
+    </table>
+    </wof-popper>
+</template>
+
+<script>
+import WofPopper from './WofPopper.vue'
+
+export default {
+  components: { WofPopper },
+    name: "WofBox",
+
+    props: {
+        title: {
+            type: String,
+            default: "Title",
+        },
+        type: {
+            type: String,
+            required: true,
+            default: "info",
+            validator(value){
+                return["error", "warning", "success", "info"].includes(value)
+            }
+        }  
+    }
+}
+</script>
+
+
+<style lang="less">
+@import './common.less';
+
+.WofBox{
+    background-color: @secondary-color;
+    min-width: 400px;
+    max-width: 600px;
+    font-size: 1.5rem;
+    word-wrap: break-word;
+    color: white;
```

Comment:
That probably should be a less attribute, i.e. @primary-neutral-color or something